starter-code/src/functions-and-arrays.js

```diff
@@ -1,93 +1,610 @@
 // Iteration #1: Find the maximum
 
+function maxOfTwoNumbers(number1, number2) {
+  if (number1 > number2) {
+    return number1;
+  } else if (number2 > number1) {
+    return number2;
+  }
+  return number1;
+}
+
 // Iteration #2: Find longest word
-const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+const words = [
+  "mystery",
+  "brother",
+  "aviator",
+  "crocodile",
+  "pearl",
+  "orchard",
+  "crackpot"
+];
+
+function findLongestWord(words) {
+  if (words.length === 0) {
+    return null;
+  }
+
+  let longestWord = words[0];
+  for (let i = 0; i < words.length; i++) {
+    if (
+      longestWord.length > words[i].length ||
+      longestWord.length === words[i].length
+    ) {
+      continue;
+    } else {
+      longestWord = words[i];
+    }
+  }
+  return longestWord;
+}
```

Comment:
This option makes a lot of sense but you didn't have to the if/ else statement.
The for loop is going to run until the end and if you dont give it nothing to do it will just ignore.
Which means that you could have checked just one thing.

```
for (let i = 0; i < words.length; i++) {
    if (longestWord.length < words[i].length) {
        longestWord = words[i]
    }
}
```
You didnt have to do the continue

You s